### ISSUE TITLE
Stop labeling X-XSS-Protection as green

### ISSUE THREAD

**Issue by drwetter** [drwetter/testssl.sh#1762]
See https://caniuse.com/?search=x-xss-protection (just stumbled over this through a discussion elsewhere, https://news.ycombinator.com/item?id=20472947)

Needs to be done also for 3.0

**Comment by keisentraut** [comment]
Most simple solution is to move it from the variable ```good_header``` to ```other_header```, see below. It will then only be rated  with severity INFO and not be shown green (light blue instead):
```
run_security_headers() {
     local good_header="X-Frame-Options X-Content-Type-Options Content-Security-Policy X-Content-Security-Policy X-WebKit-CSP Content-Security-Policy-Report-Only Expect-CT"
     local other_header="Access-Control-Allow-Origin X-XSS-Protection Upgrade X-Served-By Referrer-Policy X-UA-Compatible Cache-Control Pragma"
```

Or should it be considered a finding, if it is enabled? If yes, which severity? I can implement this, seems quite easy :)

**Comment by drwetter** [comment]
thanks @keisentraut . For 3.0: yes. For 3.1dev: The color is not a good choice, ``other_header`` should just be without any color.

While looking at it: the colons are missing too in both branches, e.g. ``X-Content-Type-Options nosniff ``--> ``X-Content-Type-Options: nosniff ``.

**Comment by keisentraut** [comment]
I fixed this for 3.1dev, please review. I have added the missing colons as well as the new header "Permissions-Policy". The header X-XSS-Protection is now rated as low severity finding. In order to avoid code duplication, I refactored the function ```run_security_headers``` a little bit.

If everything is fine, I will backport this to 3.0, too.

**Comment by keisentraut** [comment]
It now looks like this:
![securityheaders](https://user-images.githubusercontent.com/2080545/97909644-dbf43e80-1d48-11eb-95c2-5e12b71e93c2.png)
